tests/api/test_statements_get.py

```diff
@@ -144,7 +144,7 @@ def create_mock_agent(
 def insert_statements_and_monkeypatch_backend(
     request, es, mongo, clickhouse, monkeypatch
 ):
-    """Retuns a function that inserts statements into each backend."""
+    """Returns a function that inserts statements into each backend."""
```

Comment:
```suggestion
    """Return a function that inserts statements into each backend."""
```

We use imperative in Ralph